Kvent/urls.py

```diff
@@ -2,6 +2,12 @@
 
 from . import views
 
+app_name = "kvent"
 urlpatterns = [
     path('', views.index, name='index'),
+    path('accounts/createaccount', views.createaccount, name='createaccount'),
+    path('accounts/login', views.login, name='login'),
```

Comment:
This is already included in `path('accounts/', include('django.contrib.auth.urls')),`, you don't have to declare it here.